pkgs/development/python-modules/scipy/default.nix

```diff
@@ -82,7 +90,17 @@ buildPythonPackage rec {
     blas = numpy.blas;
   };
 
-  setupPyBuildFlags = [ "--fcompiler='gnu95'" ];
+  pypaBuildFlags = [
+    # Current release pins pybind11's patch version and ours is newer.
+    # Consider restoring build-time dependency check later
+    "--skip-dependency-check"
```

Comment:
remove the dependency from `pyproject.toml` instead of disabling dependency checking altogether.

Reply:
https://github.com/scipy/scipy/pull/17696

"abundance of caution", huh.